fourkitchens/suhumsci
# Shs 4916 image wysiwyg

## Summary
This fixes a WYSIWYG issue with floated images

## Steps to Test
1. Create a flexible page and add some text.
2. Insert an image, edit the media and change the floating to right. Verify it's floated right in the WYSIWYG editor and saved page.
3. Edit the page and double check left and center as well.

## Screenshots
![Screenshot 2023-05-01 at 12 31 53 PM](https://user-images.githubusercontent.com/8405274/235489183-ed6145bf-dd50-4938-aa13-7bceb03408c9.png)

## Notes
- Looking at the styles for this, I think the issue is that there's lots of overrides happening. The class that gets applied (the one that causes to image to float on the saved page) was overriding styles to make it float in the WYSIWYG. Overall, a bit of a messy situation. Hopefully in CK5 we can avoid this.
- I don't think there's a release branch on the 4K fork yet, so I just chose `develop`.
